meta-lmp-bsp/recipes-bsp/trusted-firmware-a/tf-a-fio/0007-feat-mmc-get-boot-partition-size.patch

```diff
@@ -0,0 +1,56 @@
+From 6302a29a75447522c31d43616228055b8f3f2883 Mon Sep 17 00:00:00 2001
+From: Yann Gautier <yann.gautier@st.com>
+Date: Thu, 15 Sep 2022 09:51:21 +0300
+Subject: [PATCH 2/2] feat(mmc): get boot partition size
+
+The boot partition size of an eMMC is given in ext_csd register, at
+offset 226 (BOOT_SIZE_MULT), which has to be multiplied by 128kB.
+Add a helper function mmc_boot_part_size() to get this eMMC boot
+partition size.
+
+Signed-off-by: Yann Gautier <yann.gautier@st.com>
+Change-Id: I0e8e0fc9632f147fa1b1b3374accb78439025403
```

Comment:
Its good to have the Upstream-Status with the url link of the patch.
`Upstream-Status: Backport [url]`